### PR TITLE
Jake's copyediting, including change to make sequence elements optional

### PR DESCRIPTION
**Pull request by jakebeal**

### CODE REVIEW

File: location.tex [hunk 1]
```diff
@@ -1,3 +1,4 @@
+\todo[inline]{Required sequence issue: \url{https://github.com/SynBioDex/SEPs/issues/98\#issuecomment-603569308}}
```
Comment (cjmyers) [inline location.tex]: Just place holder for now?

**Comment by cjmyers** [comment]
Looks good, but I saw one \todo still

File: location.tex (hunk 1 above)
Comment (jakebeal) [reply location.tex]: whoops; forgot to delete that when I added the sequence change.  I've done so now.

**Comment by cjmyers** [comment]
Ok, looks good to me

**Comment by jakebeal** [comment]
@cjmyers I've finished my proof-reading pass, which has added a bunch of additional modifications; please review.

**Comment by cjmyers** [comment]
Looks good.  I see you revised the provenance examples.  This was a problematic area I noticed, since the examples make less sense without XML.  I actually asked the editors about adding UML diagrams for them.  If they do get this done, we can always update the text to reflect them.

**Pull request merged**